packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableTokenImage.tsx

```diff
@@ -20,18 +22,20 @@ export const TransactionsTableTokenImage = ({
 
   const token = useMemo(() => {
     return allTokens.find(
-      t => t.symbol.toLowerCase() === tokenSymbol.toLowerCase()
+      t => t.symbol.toLowerCase() === tx.asset.toLowerCase()
     )
-  }, [allTokens, tokenSymbol])
+  }, [allTokens, tx.asset])
+
+  if (tx.assetType === AssetType.ETH) {
+    const orbitChain = orbitChains[tx.childChainId]
+
+    const logoSrc =
+      orbitChain && orbitChain.bridgeUiConfig.nativeTokenData?.logoUrl
+        ? orbitChain.bridgeUiConfig.nativeTokenData.logoUrl
+        : EthereumLogoRoundLight
 
-  if (tokenSymbol.toLowerCase() === ether.symbol.toLowerCase()) {
     return (
-      <Image
-        height={20}
-        width={20}
-        alt="ETH logo"
-        src={EthereumLogoRoundLight}
-      />
+      <Image height={20} width={20} alt="Native token logo" src={logoSrc} />
```

Comment:
Wouldn't we possibly run into the same external-site image loading issue with `nextjs/Image` tag, that we had previously (in case the native token is custom and its symbol is loaded from outside of Bridge UI)? 
We can use `img` tag here.

Reply:
good point, updated